tcms/testruns/static/testruns/js/get.js

```diff
@@ -365,22 +394,19 @@ function changeStatusBulk(statusId) {
         }], execution => {
             reloadRowFor(execution)
         })
-    });
+    })
 }
 
 function reloadRowFor(execution) {
     const testExecutionRow = $(`.test-execution-${execution.id}`)
     animate(testExecutionRow, () => {
         testExecutionRow.replaceWith(renderTestExecutionRow(execution))
 
-        treeViewBind()
+        treeViewBind(`.test-execution-${execution.id}`)
```

Comment:
This conflicts with the latest version, I still don't quite understand why it is necessary but needs rebase before doing a more detailed review.

Reply:
I fixed the conflict, but this change need to stay. Why? What is happening:
- use clicks the button
- we submit the RPC request
- we close the row and expand the next one
- RPC request succeeds
- in the success handler of the RPC request we call `renderRowFor(executon)`
- this function calls `bindEvents` which calls `treeViewBind`
- this collapses all `.list-group-item-container` elements, including the one we just opened
```js
function treeViewBind(selector = '.tree-list-view-pf') {
    // collapse all child rows
    $(selector).find(".list-group-item-container").addClass('hidden');
...
```

with this change it will only collapse the already collapsed one (the one for which we changed the status). This will not alter behaviour in other places the function is used, because the parameter has a default value